api/Sources/Api/PairQL/Dashboard/Pairs/RequestPublicKeychain.swift

```diff
@@ -0,0 +1,51 @@
+import Dependencies
+import PairQL
+
+struct RequestPublicKeychain: Pair {
+  static let auth: ClientAuth = .admin
+
+  struct Input: PairInput {
+    var searchQuery: String
+    var description: String
+  }
+}
+
+extension RequestPublicKeychain: Resolver {
+  static func resolve(with input: Input, in context: AdminContext) async throws -> PairQL
+    .SuccessOutput {
+    @Dependency(\.postmark) var postmark
+    @Dependency(\.logger) var logger
+
+    let html = """
+    <p>From admin <b>\(
+      context.admin.email
+        .raw
+    )</b> (<a href="https://gertrude-analytics.vercel.app/admins/\(
+      context.admin
+        .id
+    )">view in analytics site</a>).</p>
+    <p>They searched for "<b>\(input.searchQuery)</b>". This is what they want:</p>
+    <code>"\(input.description)"</code>
+    """
+
+    do {
+      try await postmark.send(.init(
+        to: processEnv("PRIMARY_SUPPORT_EMAIL"),
```

Comment:
can you rebase and update this to use the new postmark `.toSuperAdmin()` (not support email) method?  also, the new API's automatically log any email send failures, so that won't be necessary. but i appreciate the effort :)